webcrypto/crypto_test.go

```diff
@@ -10,10 +11,10 @@ import (
 func TestGetRandomValues(t *testing.T) {
 	t.Parallel()
 
-	ts := newTestSetup(t)
+	ts, err := newConfiguredRuntime(t)
+	require.NoError(t, err)
 
-	gotScriptErr := ts.EventLoop.Start(func() error {
-		_, err := ts.VU.Runtime().RunString(`
+	_, gotScriptErr := ts.RunOnEventLoop(`
```

Comment:
nit: This name threw me for a spin. I thought it was a boolean.

Looking at the previous code I guess this was so that the internal `err` isn't shadowing this one?

This is no longer teh case so maybe lets rename it ?

Reply:
You're right 👍🏻 I used the previous name, but with the `RunOnEventLoop` it doesn't feel fitting anymore, gonna change it 👏🏻